common/static/common/js/karma.common.conf.js

```diff
@@ -40,6 +40,9 @@
 var path = require('path');
 var _ = require('underscore');
 var appRoot = path.join(__dirname, '../../../../');
+var webpackConfig = require(path.join(appRoot, 'webpack.config.js'));
+
+delete webpackConfig.entry;
```

Comment:
karma-webpack only needs a subset of your webpack config to figure out how to bundle your files - providing entry leads to double compilation.